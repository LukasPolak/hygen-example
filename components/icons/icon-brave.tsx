import { FC } from 'react'

export const IconBrave: FC = () => {
  return <svg></svg>
}
