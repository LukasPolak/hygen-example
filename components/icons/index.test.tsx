import { render } from "@testing-library/react";

import { IconBrave } from './icon-brave'


const icons = [
  <IconBrave key="IconBrave"/>,
];

describe("Icons", () => {
  icons.forEach((icon): void => {
    it(`should render without crashing - ${icon.key}`, () => {
      render(icon);
    });

    it(`should match previous snapshot - ${icon.key}`, () => {
      const { baseElement } = render(icon);
      expect(baseElement).toMatchSnapshot(`base - ${icon.key}`);
    });
  });
});
