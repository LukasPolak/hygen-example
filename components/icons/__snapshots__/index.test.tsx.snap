// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons should match previous snapshot - IconBrave: base - IconBrave 1`] = `
<body>
  <div>
    <svg />
  </div>
</body>
`;
